src/components/Mission.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import './pages/css/Mission.css';
+import { useDispatch } from 'react-redux';
+
+import { joinMission, leaveMission } from '../redux/missions/missionsSlice';
+
+const Button = ({ isJoined, missionId }) => {
+  const dispatch = useDispatch();
+
+  if (isJoined) {
+    return (
+      <button
+        className="btn failure__button"
+        type="button"
+        onClick={() => dispatch(leaveMission(missionId))}
+      >
+        Leave Mission
+      </button>
+    );
+  }
+  return (
+    <button
+      className=" btn success__button"
+      type="button"
+      onClick={() => dispatch(joinMission(missionId))}
+    >
+      Join Mission
+    </button>
+  );
+};
+const Mission = (props) => {
+  const { mission } = props;
+
+  return (
+    <tr>
+      <td>{mission.mission_name}</td>
+      <td>{mission.description}</td>
+      <td>
+        {!mission.joined && <p className="not__member">Not a member</p>}
+        {mission.joined && <p className="active__member">Active member</p>}
+      </td>
+      <td>
+        <Button missionId={mission.mission_id} isJoined={mission.joined} />
```

Comment:
- I would suggest that the join mission button's background color not be blue but takes the background color of its parent.

Reply:
### Fixed